localnet/init/config/Settings.toml

```diff
@@ -43,6 +43,10 @@ basic_auth_password = "flip"
 # basic_auth_user = "flip"
 # basic_auth_password = "flip"
 
+[arb.rpc]
```

Comment:
again, no backup for arb? we should have  - at least an issue should be created here